src/oas/__tests__/__snapshots__/operation.test.ts.snap

```diff
@@ -516,14 +569,12 @@ Array [
     ],
     "servers": Array [
       Object {
-        "description": undefined,
+        "name": "Swagger Petstore",
```

Comment:
I forget what the logic is here - why does the server name end up getting defaulted to the name of the service again? Was there an issue or rationale back in the day that led to the behavior 🤔

Reply:
Meh, can leave for now I suppose - I'm also not sure why we do it for global servers